### ISSUE TITLE
NetworkImage widget can not parse url start with IP

### ISSUE THREAD

**Issue by jinfagang** [flutter/flutter#26647]
My app can not used outside our company, so there is a situation I have to sess image url like this:

```
Image key: NetworkImage("192.168.199.227:9000/images/chat/3ed0f8a243f831fd45f7be2c043b9d40", scale:
I/flutter ( 6025): 1.0)
```
But the widget can not get the image right while browser can parse it.

Anyway, this feature also can be done in Android and iOS, pls help me if anyone knows how let flutter make this...

**Comment by wouterhardeman** [comment]
There is no scheme in the url. Maybe adding `http://` or `https://` will help?

**Comment by zoechi** [comment]
I think this is working as intended. This is not a valid URL.
Browsers usually prefix such URLs with `http://` or `https://` automatically,
but Dart nowhere does that and I don't think this is desireable for the general case.
If you want this you can always build your own method that does that when you think it's applicable.

**Comment by jinfagang** [comment]
@wouterhardeman @zeochi Thanks for your guys kindly indicates, I will try it out!

**Comment by kangwang1988** [comment]
Based on comment above, I think this problem is resolved and not an issue for flutter.
Closed #26647
